tests/integration/publish/test_command_integ.py

```diff
@@ -90,7 +85,7 @@ def test_create_application(self):
         self.sar_client.delete_application(ApplicationId=application_id)
```

Comment:
we could probably move deletion of applications if they exist onto the teardown.

Reply:
Thanks, I moved it to tearDown.